package.json

```diff
@@ -2,25 +2,43 @@
   "name": "forkcms",
   "version": "3.6.6",
   "theme": "Bootstrap",
+  "scripts": {
+    "dev": "gulp",
+    "build": "gulp build"
+  },
   "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-autoprefixer": "^3.0.0",
-    "grunt-contrib-clean": "^0.7.0",
-    "grunt-contrib-coffee": "^0.13.0",
-    "grunt-contrib-compass": "^1.0.4",
-    "grunt-contrib-concat": "^0.5.1",
-    "grunt-contrib-copy": "^0.8.2",
-    "grunt-contrib-imagemin": "^1.0.0",
-    "grunt-contrib-jshint": "~0.11.3",
-    "grunt-contrib-nodeunit": "~0.4.1",
-    "grunt-contrib-uglify": "^0.11.0",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-fontgen": "^0.2.3",
-    "grunt-replace": "^0.11.0",
-    "grunt-svgmin": "^3.1.0",
-    "grunt-sync": "^0.5.1",
-    "grunt-usemin": "^3.1.1",
-    "grunt-webfont": "^1.1.0",
-    "load-grunt-tasks": "^3.4.0"
+    "babel-core": "^6.6.4",
```

Comment:
babel-cli isn't necessary here, as it is used through Webpack. Webpack needs babel-core and babel-loader instead.